.github/workflows/wheels.yml

```diff
@@ -157,9 +164,9 @@ jobs:
         shell: cmd
 
       - name: Upload wheels
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: dist
+          name: windows_${{ matrix.artifact_suffix }}
```

Comment:
Can we re-use `arch` and/or `cibw_arch` instead of introducing a new `artifact_suffix` variable?

Reply:
Ok, I've pushed a commit to use `arch` instead.